src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -706,6 +706,28 @@ def delete_encryption(self, bucket_name, daemon_name=None, owner=None):
     def get_encryption_config(self, daemon_name=None, owner=None):
         return CephService.get_encryption_config(daemon_name)
 
+    @Endpoint(method='GET', path='/ratelimit')
+    @EndpointDoc("Get the bucket global rate limit")
+    @ReadPermission 
+    def get_global_rate_limit(self):
+        rgwBucketRateLimit_instance = RgwRateLimit()
+        return rgwBucketRateLimit_instance.get_global_rateLimit()
+
+    @Endpoint(method='GET', path='/ratelimit')
+    @EndpointDoc("Get the bucket rate limit")
+    @ReadPermission 
+    def get_rate_limit(self, name: str ):
+        rgwBucketRateLimit_instance = RgwRateLimit()
+        return rgwBucketRateLimit_instance.get_rateLimit('bucket', name)
+    
+    @Endpoint(method='POST', path='/ratelimit')
```

Comment:
As per the design, since this is being done when you edit a bucket/user, we could just add it on the `set` endpoint of bucket and user? WDYT?
```suggestion
    @Endpoint(method='PUT', path='/ratelimit')
```

Reply:
As discussed updated the code like /rgw/user/{uid}/ratelimit for both user and bucket Get and Put